eng/pipelines/templates/pipeline.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     BuildRTM: "true"
 
   pool:
-    name: VSEng-MicroBuildVS2019
+    name: VSEngSS-MicroBuild2019
```

Comment:
Where does this pool name coming from? Is it name of VS build machine somewhere or it's something we can decide?

Reply:
The first two steps of this docs page shows how to see all the agent pools: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=azure-devops&tabs=yaml%2Cbrowser

Basically, we need to stay in communication with our Engineering Systems teams to find out what infrastructure they provide for us.